libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -271,6 +272,17 @@ class MapboxNavigation(
      */
     val historyRecorder = MapboxHistoryRecorder(navigationOptions, logger)
 
+    /**
+     * **THIS IS AN EXPERIMENTAL API, DO NOT USE IN A PRODUCTION ENVIRONMENT.**
+     *
+     * Navigation native experimental API. Do not store a reference to [Experimental] object,
+     * always access the object via `mapboxNavigation.experimental` because [MapboxNavigation] is
+     * responsible for it's lifecycle.
+     */
+    @ExperimentalMapboxNavigationAPI
```

Comment:
Could you rebased on top of `main` and use `ExperimentalPreviewMapboxNavigationAPI` instead?